fl4health/feature_alignment/tab_features_preprocessor.py

```diff
@@ -0,0 +1,147 @@
+from typing import Dict, List, Tuple
+
+import pandas as pd
+from flwr.common.typing import NDArray, Scalar
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.impute import SimpleImputer
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import MinMaxScaler, OneHotEncoder, OrdinalEncoder
+
+from fl4health.feature_alignment.string_columns_transformer import TextColumnTransformer
+from fl4health.feature_alignment.tab_features_info_encoder import TabularFeaturesInfoEncoder
+from fl4health.feature_alignment.tabular_feature import MetaData, TabularFeature
+from fl4health.feature_alignment.tabular_type import TabularType
+
+
+class TabularFeaturesPreprocessor:
+    """
+    TabularFeaturesPreprocessor is responsible for constructing
+    the appropriate column transformers based on the information
+    encoded in tab_feature_encoder. These transformers will
+    then be applied to a pandas dataframe.
+
+    Each tabular feature, which corresponds to a column
+    in the pandas dataframe, has its own column transformer. A default
+    transformer is initialized for each feature based on its data type,
+    but the user may also manually specify a transformer for this
+    feature.
+
+    Parameters
+    ----------
```

Comment:
It looks like these comments, and a few others throughout are in the "numpy" docstring format. Can you convert them over to the "Google" format? Otherwise they are really great!